src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -5996,6 +5996,74 @@ namespace NS
             End Using
         End Function
 
+        <WorkItem(41601, "https://github.com/dotnet/roslyn/issues/41601")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function SortItemsByExpandedFlag() As Task
+            Using state = TestStateFactory.CreateCSharpTestState(
+                              <Document>
+namespace NS1
+{
+    class C
+    {
+        void M()
+        {
+            $$
+        }
+    }
+
+    class MyTask1 { }
+    class MyTask2 { }
+    class MyTask3 { }
+}
+
+namespace NS2
+{
+    class MyTask1 { }
+    class MyTask2 { }
+    class MyTask3 { }
+}
+</Document>, extraExportedTypes:={GetType(TestExperimentationService)}.ToList())
+
+                Dim workspace = state.Workspace
+                workspace.TryApplyChanges(workspace.CurrentSolution.WithOptions(workspace.Options _
+                    .WithChangedOption(CompletionOptions.ShowItemsFromUnimportedNamespaces, LanguageNames.CSharp, False)))
+
+                ' trigger completion with import completion disabled
+                state.SendInvokeCompletionList()
+                Await state.WaitForUIRenderedAsync()
+
+                ' make sure expander is selected
+                state.SetCompletionItemExpanderState(isSelected:=True)
+                Await state.WaitForAsynchronousOperationsAsync()
+                Await state.WaitForUIRenderedAsync()
+
+                state.SendEscape()
+                Await state.AssertNoCompletionSession()
```

Comment:
Is this whole it relevant to this test? Or is this copy/paste from a previous test that we changed?

Reply:
This is relevant, to make sure the cache for unimported items are completely populated.